code/modules/projectiles/guns/projectile/automatic/motherfucker.dm

```diff
@@ -28,6 +28,8 @@
 	var/recentpumpmsg = 0
 	var/pumped = FALSE
 
+	gun_tags = list(GUN_SILENCABLE)
```

Comment:
It's six-barrelled. *No.* Unless you can think of a way to make it consume six of those.